keanacobarde/BEDuo
# SETUP - Test data and Initial Migration

## Description
- Created test data for API calls
- Created Initial Migration

## Related Issue
#2 

## Motivation and Context
- Required for the testing of application and future APIs

## How Can This Be Tested?
- Pull PR branch to local repo machine
- Utilize dotnet ef database update to load PostgreSQL with test data and database schema to ensure DbContext is loaded properly.

## Screenshots (if appropriate):
N / A

## Types of changes
